### ISSUE TITLE
Accuracy is low

### ISSUE THREAD

**Issue by Kiennguyen08** [Neerajj9/Text-Detection-using-Yolo-Algorithm-in-keras-tensorflow#8]
I have downloaded your pre-trained model and tested it by training 1 epoch with the same dataset, but the accuracy was low 0.25 and val_loss was at about 3.5. Was it normal?
